integration-box/cuenote-import-json/get_jobs.py

```diff
@@ -0,0 +1,74 @@
+import cuenote
+import json
+import pandas
+import time
+import os
+import sys
+from datetime import datetime
+
+os.system(f"{sys.executable} -m pip install -U pytd==1.0.0 td-client")
```

Comment:
The latest version of pytd is 1.2.0.

```suggestion
os.system(f"{sys.executable} -m pip install -U pytd==1.2.0 td-client")
```

Reply:
updated